shell/renderer/renderer_client_base.cc

```diff
@@ -194,17 +194,9 @@ void RendererClientBase::RenderThreadStarted() {
       extension_scheme);
   // In Chrome we should set extension's origins to match the pages they can
   // work on, but in Electron currently we just let extensions do anything.
-  blink::SchemeRegistry::RegisterURLSchemeAsSecure(extension_scheme);
```

Comment:
Would appreciate a second opinion on commit ecf1507 - `RegisterURLSchemeAsSecure` was removed entirely in this CL: [https://chromium-review.googlesource.com/c/chromium/src/+/2614784](https://chromium-review.googlesource.com/c/chromium/src/+/2614784) We didn't seem to be using it anywhere else and tests pass locally, but I want to make sure I'm not breaking a command line argument or extensions by removing it instead of replacing it.

Reply:
Correcting this, based on this commit: https://github.com/electron/electron/pull/27305/commits/911a288cac8bb93a5870d91e80c77732135e6546